### ISSUE TITLE
sass not ignoring partial _***.sass files

### ISSUE THREAD

**Issue by HackPoint** [angular/angular-cli#1322]
```
let appTree = new Angular2App(defaults, {
        polyfills: [
            'vendor/core-js/client/core.js',
            'vendor/systemjs/dist/system.src.js',
            'vendor/zone.js/dist/zone.js',
            'vendor/hammerjs/hammer.min.js'
        ],
        tsCompiler: {},
        vendorNpmFiles: [
            'systemjs/dist/system-polyfills.js',
            'systemjs/dist/system.src.js',
            'zone.js/dist/**/*.+(js|js.map)',
            'core-js/client/core.js',
            'es6-shim/es6-shim.js',
            'reflect-metadata/**/*.+(js|js.map)',
            'rxjs/**/*.+(js|js.map)',
            '@angular/**/*.+(js|js.map|css)',
            '@angular2-material/**/*.+(js|js.map|css)',
            'ng2-material/**/*.+(js|js.map|css)',
            'hammerjs/*.min.+(js|js.map)',
            'symbol-observable/**/*.+(js|js.map)',
        ],
        sassCompiler: {
            cacheExclude: [/\/_[^\/]+$/]
        }
    });
```

Using latest ng-cli and angular2
node 5.2.3

Webstorm11

**Comment by cDecker32** [comment]
I had a similar issue. Using cacheExclude should correctly exclude sass partials, But will also exclude them from being rebuilt when in dev mode. I suggest using the following in your build config instead.
`sassCompiler:{
        file: './src/app/**/*.scss'
      }`

**Comment by filipesilva** [comment]
Closed as issue was made obsolete by #1455.
